google-storage/src/google-content-provider.ts

```diff
@@ -0,0 +1,150 @@
+import * as Storage from '@google-cloud/storage'
+import googleUtility from './googleUtility'
+import { Observable} from "rxjs";
+//import service account details, bucketname, filename and other details from your configuration file
+import {serviceAccount,bucketName,fileName,filePathLocal} from '../config'; 
+
+// check if service account exists with correct format
+googleUtility.checkServiceAccount();
```

Comment:
This should happen inside the constructor of the content provider, not outside it.